### PR TITLE
CSI Specification compatibility matrix updated

### PR DESCRIPTION
**Pull request by Madhan-SWE**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
CSI Specification compatibility matrix updated.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,9 +6,11 @@ The IBM Power Virtual Systems Container Storage Interface (CSI) Driver provides
 
 
 # CSI Specification Compatibility Matrix
-| PowerVS CSI Driver \ CSI Version | v1.5.0 |
-| ----------------------------- | -------|
-| main branch | yes |
+| PowerVS CSI Driver | Kubernetes | CSI | Golang |
+| ----------------------------- | ----------- | -------- | -------- |
+| main | 1.24 | 1.5.0 | 1.18 |
+| 0.2.0 | 1.24 | 1.5.0 | 1.18 |
```
Comment (mkumatag) [inline README.md:12]: This is not true yet, lets hold this for now till we update required modules.

/hold

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Madhan-SWE) [reply README.md:12]: /unhold

Comment (Madhan-SWE) [reply README.md:12]: modules are updated to expected versions.

**Review by mkumatag — APPROVED** [review]
/lgtm

**Pull request merged**